airflow/hooks/hive_hooks.py

```
@@ -192,6 +192,8 @@ def get_table(self, db, table_name):
         ['state', 'year', 'name', 'gender', 'num']
         '''
         self.metastore._oprot.trans.open()
+        if not db and '.' in table_name:
+            db, table_name = table_name.split('.')
         table = self.metastore.get_table(dbname=db, tbl_name=table_name)
```

What's the result if dbname=None?, should we raise an exception if dbname is none and there is no '.'?